examples/README.md

```diff
@@ -1,40 +1,79 @@
 # Flow Examples
 
 Before continuing to the Flow examples, we recommend **installing Flow** by 
-executing the following [installation instructions](
+following the [installation instructions](
 https://flow.readthedocs.io/en/latest/flow_setup.html).
 
 The **examples** folder provides several examples demonstrating how 
-both simulation and RL-oriented experiments can be setup and executed within 
-the Flow framework on a variety of traffic problems. These examples are .py 
-files that may be executed  either from terminal or via an editor. For example,
-in order to execute the  sugiyama example in *examples/sumo*, we run:
+both non-RL simulation and RL-oriented simulatons can be setup and executed 
+within the Flow framework on a variety of traffic problems. These examples are 
+python files that may be executed either from terminal or via a text editor (or 
+Python IDE). For example, in order to execute the non-RL Ring example we run:
```

Comment:
can you run this out of IDE? since you have to pass arguments now that might not be true

Reply:
I resolved this in my commit (removed "Python IDE") @AboudyKreidieh